### DISCUSSION TITLE
[空洞骑士]我已经在等丝之鸽2了

### DISCUSSION THREAD

**Discussion by muxi0330** [share121/inter-knot#888] [其他]
⠀⢠⣴⣿⣿⣿⠷⠂⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠰⢾⣿⣿⣿⣦
⠀⣰⣿⣿⣿⡟⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠙⣿⣿⣿⣷⡀
⢰⣿⣿⣿⠏⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠘⣿⣿⣿⡇
⢸⣿⣿⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣿⣿⣿⡇
⢸⣿⣿⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣸⣿⣿⣿⡇
⠀⢻⣿⣿⣧⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣴⣿⣿⣿⡟
⠀⠀⠻⣿⣿⣿⣦⣤⣤⣤⣤⣤⣶⣶⣶⣶⣶⣤⣤⣤⣤⣶⣿⣿⣿⡿⠋
⠀⠀⠀⠈⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡿
⠀⠀⠀⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡇
⠀⠀⠀⠀⢿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡇
⠀⠀⠀⠀⢸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡿⠿⢿⣿⣿⡇
⠀⠀⠀⠀⢸⣿⡿⠋⠀⠀⠀⠙⣿⣿⣿⣿⣿⡿⠁⠀⠀⠀⠀⢹⣿⡇
⠀⠀⠀⠀⠈⣿⡇⠀⠀⠀⠀⠀⢸⣿⣿⣿⣿⡇⠀⠀⠀⠀⠀⠀⣿⣿
⠀⠀⠀⠀⠀⢹⣿⣄⠀⠀⠀⣠⣿⣿⣿⣿⣿⣿⣄⡀⠀⢀⣰⣿⡏
⠀⠀⠀⠀⠀⢀⡻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠟
⠀⠀⠀⠀⠀⠀⠙⢮⣝⠛⠿⠿⠿⠿⠿⠿⠿⠿⠿⠿⠛⠋⠁
⠀⠀⠀⠀⠀⠀⠀⠀⠙⣷⣾⢡⢾⢧⠀⢦⠈⠑⢮⣦
⠀⠀⠀⠀⠀⠀⠀⠀⠚⣿⣡⠏⢸⢸⡀⠈⢧⡀⠀⠻⣧
⠀⠀⠀⠀⠀⠀⠀⠀⣼⣷⠋⢠⣿⡄⢷⣦⠀⠙⢦⡀⠙⣧
⠀⠀⠀⠀⠀⠀⠀⢠⣿⠃⢠⡿⠃⣧⠈⠻⢧⡀⠀⠁⢦⡹⡆
⠀⠀⠀⠀⠀⠀⠀⣼⠇⣰⠏⠀⢠⠋⢷⣆⠀⠙⠶⣄⠈⢷⣧
⠀⠀⠀⠀⠀⠀⠀⣿⣠⠏⠀⣴⠋⠀⡼⠙⣆⠀⠀⠙⢷⡌⢿
⠀⠀⠀⠀⠀⠀⠀⣿⠈⢀⡾⠁⢀⡜⠁⠀⠈⠳⡄⠀⠀⠻⣼⡀
⠀⠀⠀⠀⠀⠀⠀⢹⣤⣟⣴⡴⠋⢰⠚⢳⠀⠀⣨⣦⣤⢠⢻⠗⠄
⠀⠀⠀⠀⠀⠀⠀⠈⠟⠟⠛⢧⠀⢸⠀ ⢸⠀⡇⠁⠙⠾⠏
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀ ⢆⢸ ⣇⡼

**Comment by warning-Kirov-reporting** [comment]
还是洗洗睡吧

**Comment by tianmu666** [comment]
我真幸运……

**Comment by nahida0-o** [comment]
我真幸运（乔尼音）
